.prettierignore

```diff
@@ -1 +1,6 @@
+dist
 templates
+
+# Ignore strange semicolon
+test/ts/customRunnerDBClient.ts
+test/ts/customRunnerDBUrl.ts
```

Comment:
Instead of having strange code in the repo and making a bunch of exceptions because of it, could we rather fix the code?

E.g. instead of:
```ts
(async () => {
  // ...
})()
```

have something like:

```ts
async function start() {
  // ...
}

start()
```

Reply:
@Shinigami92 could you take a look and say what is the resolution here? thanks :pray:

Reply:
Committed the improved more readable variant :+1:  
At least the naming `start` explains what's going on here :slightly_smiling_face: